### ISSUE TITLE
Typescript watch fails

### ISSUE THREAD

**Issue by SleeplessByte** [developit/microbundle#215]
Given the following files:

```json
// package.json
"main": "dist/package.js",
"umd:main": "dist/package.umd.js",
"module": "dist/package.mjs",
"source": "src/package.ts",
"devDependencies": {
  "microbundle": "^0.6.0",
  "typescript": "^3.0.3"
}
```

```Typescript
// src/package.ts
class MyPackage {
  test() { return 42 }
}

export default MyPackage
```

Running this command errors:
```Bash
$ microbundle watch

Error: ENOENT: no such file or directory, rename 
'path/to/repo/.rpt2_cache/69260a50af581442f71199615d218472c8ee4f28/code/cache_' -> 
'path/to/repo/.rpt2_cache/69260a50af581442f71199615d218472c8ee4f28/code/cache'
```

Might be related to #159 ?

No configuration files, no `tsconfig`. Note that `microbundle` (build) works just fine.

**Comment by weslleyaraujo** [comment]
I think this was fixed on https://github.com/developit/microbundle/pull/204 would be nice to have a release adding this 🙏

**Comment by stephenlautier** [comment]
I'm just trying microbundle as it looks very promising, however currently due to this it's a bit annoying during dev 🌹

**Comment by ForsakenHarmony** [comment]
https://github.com/developit/microbundle/releases/tag/0.7.0

Just published 0.7.0

Feel free to comment if this is not fixed
